src/patternfly/components/Wizard/wizard.scss

```diff
@@ -298,6 +303,7 @@
 
 .pf-c-wizard__toggle-icon {
   margin-top: var(--pf-c-wizard__toggle-icon--MarginTop);
+  transform: var(--pf-c-wizard__toggle-icon--Transform);
```

Comment:
This shouldn't be necessary, it should point down when collapsed, and up when expanded (`rotate(180deg)`)